site/_scss/blocks/_language-select.scss

```diff
@@ -0,0 +1,56 @@
+.language-select {
```

Comment:
Rather than scope all of these styles to .language-select, i think it might be worth having a `_select.scss`, similar to what web.dev does. I think we generally want all of our select elements to look like this.